### PR TITLE
[8.0] [ADD] website_sale_checkout_comment

### PR DESCRIPTION
**Pull request by pedrobaeza**
eCommerce Checkout Comments
===========================

This module extends the functionality of eCommerce to allow write conmments in checkout process.

Usage
=====

The comments writen has been stored into sale order note field.

### CODE REVIEW

File: website_sale_checkout_comment/README.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+eCommerce Checkout Comments
+===========================
+
+This module extends the functionality of eCommerce to allow write conmments in
```
Comment (atchuthan) [inline website_sale_checkout_comment/README.rst]: conmments/comments

File: website_sale_checkout_comment/controllers/main.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+                result['checkout']['note'] = data.get('note', False)
+        return result
+
+    def checkout_form_save(self, checkout):
+        res = super(WebsiteSale, self).checkout_form_save(checkout)
+        if 'note' in checkout:
+            order = request.website.sale_get_order(
+                force_create=1, context=request.context)
+            order.note = checkout['note']
+        return res
```
Comment (atchuthan) [inline website_sale_checkout_comment/controllers/main.py]: website_sale_checkout_comment/controllers/main.py:22:19: W292 no newline at end of file

File: website_sale_checkout_comment/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "category": "Website",
+    "website": "http://www.tecnativa.com",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "website_sale",
+    ],
+    "data":[
```
Comment (atchuthan) [inline website_sale_checkout_comment/__openerp__.py]: website_sale_checkout_comment/__openerp__.py:17:11: E231 missing whitespace after ':'

**Review by atchuthan — APPROVED** [review]
Code review :+1: except some minor changes

**Comment by pedrobaeza** [comment]
Please squash on merge.

**Review by susport — COMMENTED** [review]
Code review and runbot 👍

**Review by lasley — APPROVED** [review]
Code looks good. Any possibility of tests?

**Comment by pedrobaeza** [comment]
Well, this time we won't provide a test as it requires a full tour just for this little option. Merging as I'm not the original author.

**Pull request merged**